src/js/components/organisms/brands/brands2.scss

```diff
@@ -0,0 +1,44 @@
+.brands2 {
+    padding-top: $buildingUnit * 4;
+
+    .list-unordered-limit {
+        ul {
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: center;
+            list-style: none;
+
+            margin-top: -$buildingUnit;
+            margin-bottom: 0;
+            margin-left: auto;
+            margin-right: auto;
+            padding-left: $buildingUnit;
+
+            max-width: 6 * (120 + $buildingUnit) + $buildingUnit;
```

Comment:
Changing the image height will kill the row animation. Maybe convert the magic number to a constant